frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -10,6 +10,7 @@ export const getAppLabels = (
     'app.kubernetes.io/instance': name,
     'app.kubernetes.io/component': name,
     'app.kubernetes.io/name': imageStreamName,
+    'app.openshift.io/runtime': imageStreamName,
```

Comment:
@divyanshiGupta I don't believe `getAppLabels` is run when rendering an icon. I can't find any path of execution while it's rendering the topology view.

Is this an additional, unrelated, change? Or am I missing something?

Reply:
@andrewballantyne `getAppLabels` is used in import flows for setting the default labels in the resources so that the resources can be created with the desired initial labels. In this PR we were using `runtime` label for `builderImage` icon but it was not being set anywhere. Hence this change was necessary.